kontrakter/politi/innsettelsesordre/1.0/innsettelsesordre.schema.json

```diff
@@ -127,29 +144,53 @@
       "required": ["kjennelse"],
       "additionalProperties": false,
       "properties": {
-        "merknad": { "type": "string" },
-        "kjennelse": { "type": "string" }
+        "merknad": {
+          "type": "string"
+        },
+        "kjennelse": {
+          "type": "string"
+        }
       }
     },
+    "paagripelse": {
+      "type": "object",
+      "description": "Tid og sted for pågripelse",
+      "properties": {
+        "tidspunkt": {
+          "$ref": "#/definitions/dateTime"
+        },
+        "sted": {
```

Comment:
paagripelse -> sted. Ideelt sett minimum required slik at man får med NOE som de kan bruke for å jakte ned denne informasjonen i etterkant.